### PR TITLE
I386 update run tutorial to use tifs

### PR DESCRIPTION
**Pull request by tst-eclamar**
Update run_tutorial.sh to use TIFs instead of JP2s

### CODE REVIEW

**Comment by tst-lsavoie** [comment]
It looks like this needs to be rebased.  It includes commits from other PRs.

File: earth_enterprise/src/fusion/tools/gee_test/README [hunk 1]
```diff
@@ -1,58 +1,102 @@
...
+
+You must also add source volumes with the names "src" and "tutorial", respectively:
 /usr/local/google/gevol_test/src
 /opt/google/share/tutorials/fusion
 
 sudo /opt/google/bin/geselectassetroot --assetroot  /usr/local/google/gevol_test/assets
 
+sudo /opt/google/bin/geconfigurepublishroot --path=/usr/local/google/gevol_test/published_db
+
+You can check your asset root and source volumes using:
```
Comment (tst-ccamp) [inline earth_enterprise/src/fusion/tools/gee_test/README]: You should add an additional step needed to run with 5.1.3 to copy the tif files into the /opt/google/share/tutorial/fusion directory

File: earth_enterprise/src/fusion/tools/gee_test/README [hunk 2]
```diff
@@ -28,6 +28,11 @@ sudo /opt/google/bin/geselectassetroot --assetroot  /usr/local/google/gevol_test
 
 sudo /opt/google/bin/geconfigurepublishroot --path=/usr/local/google/gevol_test/published_db
 
+You will need to copy several TIFs to the Imagery and Terrain directories
+(it's simplest to unpack the entire tutorial tar):
+  wget http://data.opengee.org/FusionTutorial-Full.tar.gz
+  sudo tar -zxf FusionTutorial-Full.tar.gz -O /opt/google/share/tutorials/fusion
```
Comment (tst-lsavoie) [inline earth_enterprise/src/fusion/tools/gee_test/README]: I think this -O needs to be a -C.  The -O failed for me on RHEL 7.

Comment (tst-eclamar) [inline earth_enterprise/src/fusion/tools/gee_test/README]: Quite right - got confused with wget.

**Pull request merged**